buildah_test.go

```diff
@@ -40,7 +40,6 @@ func TestMain(m *testing.M) {
 }
 
 func TestOpenBuilderCommonBuildOpts(t *testing.T) {
-	t.Parallel()
```

Comment:
When you remove this again would it make sense to add a comment to each here why they cannot be parallelized. It would help someone in the future wondering why this on is not parallel or prevent someone from trying to added parallel here, same in the other places hwere you had to revert this.

Reply:
Sure.